docs/layers/identity/identity.mdx

```diff
@@ -25,110 +29,142 @@ demonstrating how it can be used to establish fine-grained access control for an
+To simplify managing IAM access roles, we’ve created two concepts. AWS Teams and AWS Team Roles.
 
-Now that we have the roles created with Teams and Team Roles, lets put it to work.
+All access in AWS is “Team” oriented. A “Team” is a group of users that we deploy into a central “identity” account. Then we deploy a number of “Team Roles” into every single account. A “Team” can assume any “Team Role” across the organization.
 
-We need a tool that solves many problems listed above while being Multi-OS compatible and ideally programmable. We have
-tried several tools, the one that works very well and appears to be gaining exponential traction is
-[Leapp](https://www.leapp.cloud/).
+We’ll explain in detail in the following pages.
 
-Leapp is a Desktop Application (and has a CLI) that manages your active sessions for your AWS Config. This means it
-handles logging in, and assigning those credentials generated to a given AWS profile.
+### Highly Accessible
 
-Before continuing on all the problems Leapp solves, we need to state that the AWS Configuration file (`~/.aws/config`)
-is usually highly dependent on your sign-in method. Different sign-in methods use different profile configurations. The
-beauty of using Leapp is we can have a single config file regardless of your sign-in method. I will discuss this in the
-next section.
+Then to make it incredibly easy for users to log in, we recommend using Leapp to manage local AWS sessions. Leap supports all Identity Providers and can automatically refresh credentials when they expire.
 
-So Leapp manages our credentials. It does so in a couple beautiful ways.
+Plus, it’s open source and free to use. It also offers paid options that make it easier for administrators to share configurations. Please note, the Cloud Posse solution does not require any subscription.
 
-1. It works with any Identity Provider, and since it's an Electron application, it can prompt for sign-in, in the app or
-   in the browser of choice.
-2. It automatically refreshes your AWS credentials before they expire (No mid-terraform apply failures from expired
-   credentials)
-3. If we are using AWS IAM Identity Center (SSO) it will sync the available `PermissionSets` from your login as
-   available sessions.
-4. It's Open Source
+### Pre-Configured for your Team
 
-Since we auto-generate an AWS Configuration file and Leapp can use a CLI to create Sessions that manage Profile
-credentials, we can set up automated AWS access for AWS IAM Identity Center or AWS SAML.
+Finally, to get everyone up and running on your Team, we pre-configured a single, common AWS config for all users.
 
-### Easily Configured: Identity Provider independent Configuration and Sign-in.
+We preinstall the AWS config in the Geodesic toolbox, and users will automatically assume the correct access roles in AWS. Users can quickly apply Terraform across accounts and assume roles directly in the accounts they have access.
```

Comment:
"users will automatically assume the correct access roles in AWS"

Why? ...link to dynamic roles

Reply:
this works with or without dynamic roles. with dynamic roles you can automatically assume terraform or planner, but without you still can apply where you have access and simply get an access failed message where you dont have access

Reply:
I'll add a better note